.travis.yml

```diff
@@ -29,4 +29,4 @@ before_script:
   - cd ~/builds/ss
 
 script:
-  - phpunit silverstripe-lumberjack/tests/
\ No newline at end of file
+  - vendor/bin/phpunit lumberjack/tests/
```

Comment:
I'd changed the vendor/projectname on this fork, which is what generates the actual install path.